lib/schemas.ts

```diff
@@ -1,10 +1,10 @@
-import { z } from 'zod';
+import { ZodType, z } from 'zod';
 
-export const LoginSchema = z.object({
+export const LoginSchema: ZodType<UserLogin> = z.object({
   email: z.string().email({
     message: 'Please enter your email address',
   }),
-  password: z.string().min(1, {
+  password: z.string().min(2, {
```

Comment:
why change to 2? now when user types 1 character into password field and tries to log in the error won't show up

Reply:
Em, what? I changed it for 2, because in my test when i type "1" in password input, login was successful. It's for test only, but in future this should be like 8 characters minimum

Reply:
no it should not its not a registration form, only a login form